packages/mongo-nav/src/project-nav/ProjectNav.spec.tsx

```diff
@@ -138,6 +141,21 @@ describe('packages/mongo-nav/src/project-nav', () => {
     });
   };
 
+  const testForProjectStatusBadge = (isVisible: boolean) => {
+    it(`${
+      isVisible ? 'displays' : 'does not display'
+    } the project status badge in the nav`, () => {
+      const statusBadge = expectedElements.projectStatusBadge;
+
+      if (isVisible) {
+        expect(statusBadge).toBeInTheDocument();
+        expect((statusBadge as Element).innerHTML).toContain('ACTIVE');
+      } else {
+        expect(statusBadge).toBeNull();
```

Comment:
I've been writing some jest tests and discovered you can write instead: `expect(statusBadge).not.toBeInTheDocument();`

Reply:
👍